README.md

```diff
@@ -40,6 +40,28 @@ class MyModel(model.Model):
     color = ColorField(default='#FF0000')
 ```
 
+### Color Format
+ColorField defaults to HEX format but also support HEXA. To set the format:
+
+```python
+from colorfield.fields import ColorField
+from django.db import models
+
+class MyModel(model.Model):
+    color = ColorField(color_format='hexa')
+```
+
+### Color Palette
+You can pass a color palette to add quick access to specific colors to the widget:
```

Comment:
It would be nice to add a screenshot for the palette, you can use #49 to upload an image and get its url.

Reply:
added